DTS-STN/secure-client-hub
# Update auth-dev to support credentialsProvider

### Description of proposed changes:
This PR updates auth-dev to include credentialsProvider again for testing purposes. I've added some checks based on the type of provider in the active session for the `ValidateSession` calls. Added a check on the index page that redirects to the dashboard if there is an active session. Changed the redirect function in our auth helpers back to `/api/auth/signin` as that is the default next-auth page, otherwise it would redirect to our custom login page which causes issues.

### What to test for/How to test
1. Pull in branch
2. Ensure `AUTH_DISABLED` is set to false
3. To login to credentials provider, select a language on the splash page and when redirected to the login form, enter `test1, password1` in the respective login fields
4. Click `Sign in with Credentials`
5. You should land on the dashboard
6. Ensure that the `refresh-msca` calls are returning 200
7. Sign out
8. Turn proxy on and click `Sign in with ECAS`, you should be redirected to the stream links page. Confirm the rest of the app works as expected.
